src/nwb_benchmarks/setup/_reduce_results.py

```diff
@@ -39,9 +39,7 @@ def reduce_results(machine_id: str, raw_results_file_path: pathlib.Path, raw_env
     parsed_environment_info = _parse_environment_info(raw_environment_info=raw_environment_info)
     environment_id = get_dictionary_checksum(dictionary=parsed_environment_info)
 
-    # In timestamp, replace separators with underscores for file path
-    unix_time_to_datetime = str(datetime.datetime.fromtimestamp(raw_results_info["date"] / 1e3))
-    timestamp = unix_time_to_datetime.replace(" ", "-").replace(":", "-")
+    timestamp = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
```

Comment:
This also fixes a bug where none of my latest results were being posted because they all shared the same timestamp from ASV (no idea why) and so were overwriting data to the same file